src/doc/python_scripting/functions.rst

```diff
@@ -1560,36 +1568,29 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineCurveExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
     DefineCurveExpression creates new curve variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
+    Curve variables are a collection of X - Y coordinates that form a curve.
+    Curve variables are used in the Curve, Parallel Coordinates, Scatter and Spreadsheet plots.
+
+    The variableName argument is a string that contains the name of the new variable.
+    You can pass the name of an existing expression if you want to provide a new expression definition.
+    The expression argument is a string that contains the definition of the new variable in terms of `built-in expressions <../using_visit/Quantitative/Expressions.html#built-in-expressions>`_ and pre-existing variable names using VisIt_'s `expression grammar <../using_visit/Quantitative/Expressions.html#expression-grammar>`_.
 
 
 **Example:**
 
 ::
 
   #% visit -cli
-  OpenDatabase("/usr/gapps/visit/data/globe.silo")
-  DefineScalarExpression("myvar", "sin(u) + cos(w)")
-  # Plot the scalar expression variable.
-  AddPlot("Pseudocolor", "myvar")
-  DrawPlots()
-  # Plot a vector expression variable.
-  DefineVectorExpression("myvec", "{u,v,w}")
-  AddPlot("Vector", "myvec")
+  OpenDatabase("/usr/gapps/visit/data/lines.curve")
+  DefineCurveExpression("myvar", "2 * curve1")
```

Comment:
I am not sure I understand this curve expression but it seems like its a scalar multiple of an *existing* curve expression. How does one construct a curve expression from an x variable and a y variable?

Reply:
It multiplies the y values by 2. I added a comment to that effect. I don't know that we have a way to create a curve from an x and y variable.